.gitignore

```diff
@@ -123,3 +123,14 @@ port/omrsyslogmessages.h
 port/omrsyslogmessages.rc
 port/MSG00001.bin
 x64/
+
+# Test Output
+*.dylib.dbg
```

Comment:
.dylib.dbg is not a test output file.  I suggest moving this up to be with the .dylib entry.